keyboards/crkbd/keymaps/thunderbird2086/oled.c

```diff
@@ -0,0 +1,304 @@
+// Copyright 2021 Allen Choi (@thunderbird2086)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+#include "keycodes.h"
+
+
+// 5x3 Logos
+void render_qmk_logo(void) {
+    static const char PROGMEM font_qmk_logo[16] = {
+        0x80, 0x81, 0x82, 0x83, 0x84,
+        0xa0, 0xa1, 0xa2, 0xa3, 0xa4,
+        0xc0, 0xc1, 0xc2, 0xc3, 0xc4,
+        0
+    };
+    oled_write_P(font_qmk_logo, false);
+};
+
+// 5x2 Keyboard, Controller logos
+void render_kb_split(void) {
+    static const char PROGMEM font_kb_split[11] = {
+        0xb5, 0xb6, 0xb7, 0xb8, 0xb9,
+        0xd5, 0xd6, 0xd7, 0xd8, 0xd9,
+        0
+    };
+    oled_write_P(font_kb_split, false);
+};
+
+// 5x1 Layer indicator
+
+void render_layer(void) {
+    static const char PROGMEM font_layer[4][6] = {
+        {0x9a, 0x9b, 0x9c, 0x9d, 0x9e, 0},
+        {0xba, 0xbb, 0xbc, 0xbd, 0xbe, 0},
+        {0xda, 0xdb, 0xdc, 0xdd, 0xde, 0},
+        {0x95, 0x96, 0x97, 0x98, 0x99, 0},
+    };
+
+    uint8_t layer = 0;
+    if (layer_state_is(_FUNCTION)) {
+        layer = 1;
+    } else if (layer_state_is(_CODE)) {
+        layer = 2;
+    } else if (layer_state_is(_ADJUST)) {
+        layer = 3;
+    }
+    oled_write_P(font_layer[layer], false);
+};
+
+
+void render_layers(void) {
+    static const char PROGMEM font_layers[4][16] = {
+        {0x20, 0x85, 0x86, 0x87, 0x20,
+         0x20, 0xa5, 0xa6, 0xa7, 0x20,
+         0x20, 0xc5, 0xc6, 0xc7, 0x20,
+         0},
+        {0x20, 0x88, 0x89, 0x8a, 0x20,
+         0x20, 0xa8, 0xa9, 0xaa, 0x20,
+         0x20, 0xc8, 0xc9, 0xca, 0x20,
+         0},
+        {0x20, 0x8b, 0x8c, 0x8d, 0x20,
+         0x20, 0xab, 0xac, 0xad, 0x20,
+         0x20, 0xcb, 0xcc, 0xcd, 0x20,
+         0},
+        {0x20, 0x8e, 0x8f, 0x90, 0x20,
+         0x20, 0xae, 0xaf, 0xb0, 0x20,
+         0x20, 0xce, 0xcf, 0xd0, 0x20,
+         0},
+    };
+
+    uint8_t layer = 0;
+    if (layer_state_is(_FUNCTION)) {
+        layer = 1;
+    } else if (layer_state_is(_CODE)) {
+        layer = 2;
+    } else if (layer_state_is(_ADJUST)) {
+        layer = 3;
+    }
+    oled_write_P(font_layers[layer], false);
+};
+
+#if defined(RGB_MATRIX_ENABLE) || defined(RGBLIGHT_ENABLE)
+
+#if defined(RGBLIGHT_ENABLE)
+extern rgblight_config_t rgblight_config;
+#endif
+
+#if defined(RGB_MATRIX_ENABLE)
+extern rgb_config_t rgb_matrix_config;
+#endif
```

Comment:
There are functions for getting the values for these, and these shouldn't be used directly. 

https://docs.qmk.fm/#/feature_rgblight?id=query
https://docs.qmk.fm/#/feature_rgb_matrix?id=query-current-status